src/Tribe/Admin/Settings.php

```diff
@@ -224,6 +234,37 @@ public function add_admin_pages() {
 		$this->maybe_add_app_shop();
 	}
 
+	/**
+	 * Redirects users from an outdated help page to the updated Help Hub page in the WordPress admin.
+	 *
+	 * Checks the `page` and `post_type` query parameters, and if they match the old help page slug.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function redirect_to_help_hub(): void {
+		$page      = tribe_get_request_var( 'page' );
+		$post_type = tribe_get_request_var( 'post_type' );
+
+		// Exit if the request is not for the old help page.
+		if ( Plugin::POSTTYPE !== $post_type || 'tec-events-help' !== $page ) {
```

Comment:
where is `tec-events-help` defined? can we move that one to a prop as well?

Reply:
It was originally hard-coded in the file and was never stored. I moved it into `$original_help_slug`.